pandas/io/pytables.py

```diff
@@ -1441,9 +1439,11 @@ def error(t):
             if value is None:
 
                 _tables()
-                if getattr(group, "table", None) or isinstance(
-                    group, _table_mod.table.Table
-                ):
+                # mypy can't tell that _table_mod is not None, so we have
+                #  to do a type: ignore
+                cond1 = getattr(group, "table", None)
+                cond2 = isinstance(group, _table_mod.table.Table)  # type: ignore
```

Comment:
the message is `error: Item "None" of "Optional[Any]" has no attribute "table"`

`assert _table_mod is not None` (and ensuring the correct type for _table_mod) is preferable to ignore.

Reply:
that assertion made the rest unnecessary, good idea